text/0037-minting.md

```diff
@@ -0,0 +1,167 @@
+    * Signer set: the set of _N_ signers, `{A_i, A_{i+1}, ..., A_N}`
+    * Threshold: the number, _M_, of signers required per `MintTx`
+    * Mint limit: the total amount that can be minted in this configuration
+* the Ed25519 multisig signature set, including the total _M_ signatures required for the _M-of-N_ threshold, specified by the governors configuration, previously signed by the Foundation minting trust root key.
+
+
+All of the values above are committed to the blockchain and will be used when verifying `MintTxs`.
+
+Consensus validator nodes perform consensus when a `MintConfigTx` is proposed via a client RPC endpoint on the nodes. This provides flexibility to the minting governors and keeps the network resilient so that a quorum must be reached to allow a new minting configuration, while ensuring liveness of the network, so that the network need not restart in order to change the minting configuration.
+
+Consensus validator nodes use the latest minting configuration to validate `MintTxs`.
+
+* Note, however, that nodes in catchup mode do not participate in consensus. The network is already resilient to nodes in catchup, and the minting procedures are similarly resilient.
+
+## Governors: The Signer Set for a Consensus Validator
+[governors]: #governors
+
+The consensus validator nodes must maintain a set of "master minters," also known as _governors_, who can sign the `MintConfigTx` transactions. The governor set is maintained in the node's startup configuration file, and it is the responsibility of the node operator to perform due diligence on any governor they are adding to the set.
+
+The governor set is a set of Ed25519 public keys and a threshold, and a `MintConfigTx`'s `TxPrefix` must be signed by the threshold number of governors required by each node in a quorum of nodes to establish a new Minting Configuration for the network.
+
+To further protect the governors configuration, and to ensure all nodes are running with identical configuration, the consensus enclave contains a baked-in Ed25519 public key called the `Minting Trust Root` key. The consensus enclave requires that the governors configuration be signed by the matching private key (held by the MobileCoin Foundation), and that the signature is provided upon initialization. Without a valid signature over the governors configuration, the consensus enclave will refuse to initialize.
+
+The governors configuration is also hashed into the node's responder ID, preventing node-to-node attestation if the configuration differs. This is similar to how minimum fee configuration is currently handled.
+
+### Multiple Asset Fees
+
+The startup-config for the consensus validator node contains the governors, as well as a `TokensConfig`, which specifies the following per token:
+
+* Token ID
+* Minimum fee
+* Whether any fee is allowed 
+* The list of governors who can change the Minting Config for this token (issue `MintConfigTx` transactions)
```

Comment:
TokenIds config can be changed post mint? What can be changed about them?

Reply:
Not sure what you're asking since I don't know what you're referring to by saying `TokenIds config can change post mint`.
The governors are the signers that can issue `MintConfigTx` transactions, and those transactions determines who can issue `MintTx` transactions. A `MintTx` transaction that gets accepted results in a mint (generation of new `TxOut`s). Following that, the governors can put in place a new `MintConfigTx` that changes who can issue new `MintTx`s. Previous ones that were already accepted into the blockchain are unaffected.